minions/async/src/main/java/io/github/microcks/minion/async/producer/MQTTProducerManager.java

```diff
@@ -45,6 +49,9 @@
    @ConfigProperty(name = "mqtt.server")
    String mqttServer;
 
+   @ConfigProperty(name = "mqtt.clientid", defaultValue="microcks-async-minion")
+   String mqttClientId;
+
```

Comment:
Could you explain the rationale with making the ClientId configurable? I'd be happy to learn about that point and use-cases.

Reply:
This is to resolve a problem I ran into - I ended up with two microcks instances connecting to the same broker. The re-use of the same client id caused random disconnections of the two MQTT client connections. Being able to set this as a property allowed me to work around this problem.

Reply:
Ok thanks.